chunking_list/chunking_list.py

```diff
@@ -0,0 +1,13 @@
+def chunk_list(list_1, chunk_size):
+    chunks = []
+    chunk = []
+    for index, item in enumerate(list_1):
+        chunk.append(item)
+        if (index + 1) % chunk_size == 0:
+            chunks.append(chunk)
+            chunk = []
+    if chunk:
+        chunks.append(chunk)
+
+    return chunks
+
```

Comment:
Additionally, provide code to test your function using `assert` statements, drawing examples from the task description.
For instance:
```
# Test case 1
items1 = ['a', 'b', 'c', 'd', 'e', 'f']
chunk_size1 = 2
assert chunk_list(items1, chunk_size1) == [['a', 'b'], ['c', 'd'], ['e', 'f']]
```